Makefile

```diff
@@ -290,3 +295,35 @@ $(GOPATH)/bin/gomod2rpmdeps:
 	@sed -e 's/__VERSION__/'$(CRC_VERSION)'/g' \
 	     -e 's/__OPENSHIFT_VERSION__/'$(BUNDLE_VERSION)'/g' \
 	     $< >$@
+
+$(HOST_BUILD_DIR)/split: packaging/windows/split.go
+	go build -o $(HOST_BUILD_DIR)/split packaging/windows/split.go
+
+$(PACKAGE_DIR)/product.wxs: $(PACKAGE_DIR)/product.wxs.in
+
+msidir: BUNDLE = crc_hyperv_$(BUNDLE_VERSION).$(BUNDLE_EXTENSION)
+msidir: clean $(HOST_BUILD_DIR)/crc-embedder $(HOST_BUILD_DIR)/split $(BUILD_DIR)/windows-amd64/crc.exe check_bundledir $(PACKAGE_DIR)/product.wxs
+	mkdir -p $(PACKAGE_DIR)/msi
+	$(HOST_BUILD_DIR)/crc-embedder embed --goos windows --log-level debug --exclude-bundle $(BUILD_DIR)/windows-amd64/crc.exe
+	cp $(HOST_BUILD_DIR)/crc.exe $(PACKAGE_DIR)/msi/crc.exe
+ifeq ($(MOCK_BUNDLE),true)
+	touch $(PACKAGE_DIR)/msi/$(BUNDLE).0 $(PACKAGE_DIR)/msi/$(BUNDLE).1 $(PACKAGE_DIR)/msi/$(BUNDLE).2
+else
+	cp $(HYPERV_BUNDLENAME) $(PACKAGE_DIR)/msi/
+	$(HOST_BUILD_DIR)/split $(PACKAGE_DIR)/msi/$(BUNDLE)
+	rm $(PACKAGE_DIR)/msi/$(BUNDLE)
+endif
+	cp -r $(PACKAGE_DIR)/Resources $(PACKAGE_DIR)/msi/
+	cp $(PACKAGE_DIR)/*.wxs $(PACKAGE_DIR)/msi
+	rm $(PACKAGE_DIR)/product.wxs
+
+$(BUILD_DIR)/windows-amd64/crc-windows-amd64.msi: msidir
+	pwsh -NoProfile -Command "candle.exe -o $(PACKAGE_DIR)/msi/ $(PACKAGE_DIR)/msi/*.wxs"
+	pwsh -NoProfile -Command "cd $(PACKAGE_DIR)/msi; light.exe -ext WixUIExtension -ext WixUtilExtension -sacl -spdb -sice:ICE61 -out ../../../$@ *.wixobj"
+
+CABS_MSI = "cab1.cab,cab2.cab,cab3.cab,crc-windows-amd64.msi"
```

Comment:
The number of files seem hardcoded in various place, could get cumbersome/error-prone when we need to change the number of files :-/ Not sure how to make things better though

Reply:
So let's move this as an improvement for the next set of changes.